packit-ci.fmf

```diff
@@ -35,6 +35,11 @@
      - /functional/db-mysql-sanity-on-localhost
      - /functional/tenant-allowlist-sanity
      - /functional/measured-boot-swtpm-sanity
+     # now set zeromq as a default revocation notifier and test it
+     - /setup/configure_default_revocation_notifier/zeromq
+     - /functional/basic-attestation-on-localhost
+     # now default revocation notifier back to agent
+     - /setup/configure_default_revocation_notifier/agent
```

Comment:
Would be required to call again `/functional/basic-attestation-on-localhost` after this line?

Reply:
It won't. All tests on lines `25-37` are run with the default (currently `agent`) notifier.
This is just to restore the original configuration prior running upstream tests.

Reply:
Ouch yes, now I see the line before